src/components/Steps/Criteria/custom.js

```diff
@@ -50,46 +59,74 @@ class CustomCriteria extends React.Component {
+                        value={url}
+                        data-tip={this.state.urlError || ''}
+                        onBlur={({ target: { value } }) => {
+                            this.setState({ urlError: this.validateUrl(value) });
+                        }}
+                        onChange={({ target: { value } }) => {
+                            this.setState({ urlError: this.validateUrl(value) });
+                            this.props.updateCriteriaData({ url: value });
+                        }}
+                    />
+                </div>
+                <div className="line">
+                    <label>Property access</label>
+                    <input
+                        className={classnames('input', { error: this.state.propertyAccessError })}
+                        value={propertyAccess}
+                        data-tip={this.state.propertyAccessError || ''}
+                        onBlur={({ target: { value } }) => {
+                            this.setState({
+                                propertyAccessError: this.validatePropertyAccess(value),
+                            });
+                        }}
+                        onChange={({ target: { value } }) => {
+                            this.setState({
+                                propertyAccessError: this.validatePropertyAccess(value),
+                            });
+                            this.props.updateCriteriaData({ propertyAccess: value });
+                        }}
+                    />
+                </div>
+                <div className="line">
+                    <label>Max rating value</label>
+                    <input
+                        className={classnames('input', { error: this.state.maxRatingValueError })}
+                        value={maxRatingValue}
+                        data-tip={this.state.maxRatingValueError || ''}
+                        onBlur={({ target: { value } }) => {
+                            this.setState({
+                                maxRatingValueError: this.validateMaxRatingValue(value),
+                            });
+                        }}
+                        onChange={({ target: { value } }) => {
+                            this.setState({
+                                maxRatingValueError: this.validateMaxRatingValue(value),
+                            });
+                            this.props.updateCriteriaData({ maxRatingValue: value });
+                        }}
+                    />
+                </div>
+                <div className="line">
+                    <label>Importance</label>
+                    <InputRange
+                        className="input"
+                        minValue={1}
+                        maxValue={100}
+                        step={1}
+                        value={importance}
+                        onChange={imp => this.props.updateCriteriaData({ importance: imp })}
```

Comment:
importance instead of imp wouldnt be better?

Reply:
I had to use imp since importance is already defined in upper scope (ESLint).